Result.xcodeproj/project.pbxproj

```diff
@@ -458,6 +456,7 @@
 				INFOPLIST_FILE = Result/Info.plist;
 				INSTALL_PATH = "@rpath";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.antitypical.$(PRODUCT_NAME:rfc1034identifier)";
```

Comment:
From the release notes

>  The new Product Bundle Identifier build setting (PRODUCT_BUNDLE_IDENTIFIER) is the recommended place to set the Bundle Identifier for a target. The target's Info.plist should be configured to use this build setting by referencing it as "$(PRODUCT_BUNDLE_IDENTIFIER)" in the value for the CFBundleIdentifier key. Xcode will offer to configure this for you when you accept the "Upgrade to recommended settings" project modernization in the issue navigator, unless your target preprocesses its Info.plist, in which case you will need to configure this manually. This change is backwards-compatible to older versions of Xcode. This change is required to make certain features work, such as On Demand Resources, if your target preprocesses its Info.plist. (20887827)

This is what the default Xcode conversion did. I'll see if I can bump this up a level.